app/controllers/users_controller.rb

```diff
@@ -1,5 +1,12 @@
 class UsersController < ApplicationController
-  before_action :load_user, only: :show
+  before_action :load_user, except: %i(new create index)
+  before_action :logged_in_user, except: %i(show new create)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: :destroy
+
+  def index
+    @users = User.sort_by_name.page params[:page]
```

Comment:
please add the 'per' argument for this method

Reply:
![image](https://user-images.githubusercontent.com/68467813/91680971-9320dd80-eb7f-11ea-9c34-f89079ba9f0f.png)
I had set the default page in kaminari_config.rb, do i still need to add on "per" argument in this file